docs/spec/nmt.md

```diff
@@ -167,16 +168,89 @@ This is the same as the queried range.
+The namespace `03` is the smallest namespace larger than `02`. 
+By traversing the branch from the leaf with namespace `03` to the root, we find a node with hash `03 04 52c7c03` whose namespace range doesn't overlap with `02`. 
+This node is the highest such node along the branch.
+The `SubtreeHash` is the hash of that node, which is `03 04 52c7c03`.
+The `start` and `end` indices indicate its position in the respective level. 
+In this case, `start = 1` and `end = 2`. 
+Note that node indices start at `0` from left to right at each level.
+The `nodes` form the index-based Merkle inclusion proof of the `SubtreeHash` to the tree root `T`. 
+The `nodes` set includes `00 00 ead8d25`, the left sibling of `03 04 52c7c03`.
+
+In summary, the short namespace absence proof for `NS = 02` in this tree consists of `SubtreeHash = 03 04 52c7c03`, `start = 1`, `end = 2`, and the `nodes` set containing `00 00 ead8d25`.
+
+```markdown
+                                 00 04 b1c2cc5                                Tree Root
+                           /                       \
+                          /                         \
+                        NsH()                       NsH()
+                        /                             \
+                       /                               \
+                      /                           -------------
+               00 00 ead8d25                     |03 04 52c7c03|              Non-Leaf Nodes
+                /         \                       -------------
+              /            \                    /               \
+            NsH()          NsH()              NsH()             NsH()
+            /                \                /                   \
+    00 00 5fa0c9c       00 00 52385a0    03 03 71ca46a       04 04 b4a2792    Leaf Hashes
+        |                   |                 |                   |
+      NsH()               NsH()              NsH()               NsH()
+        |                   |                 |                   |
+00 6c6561665f30      00 6c6561665f31    03 6c6561665f32      04 6c6561665f33  Leaves with namespaces
+
+        0                   1                  2                  3           Leaf Indices
+```
+
+#### Short Namespace Absence Proof Verification
+
+A short namespace absence proof is deemed valid if it meets the following:
+
+1) The minimum namespace of the `SubtreeHash` in the `proof` is greater than the queried `NS`.
+It is important to note that there can be multiple candidates, each belonging to a different level of the tree, that satisfy this requirement for the `SubtreeHash`.
+All such candidates are deemed valid from the verification perspective.
+2) The verification of Merkle inclusion proof of the returned `SubtreeHash` is valid.
+3) The `proof` satisfies the proof completeness as explained in the [Verification of NMT Inclusion Proof](#verification-of-nmt-inclusion-proof).
```

Comment:
> Thanks for the explanation yesterday. I think it's not so intuitive why you only need one SubTreeHash i.e. that the values in nodes which represent the proof path provides the other side necessary to prove that the namespace does not exist in the tree.

@cmwaters regarding this comment you made, I think you are referring to the completeness property,  it should be verified on the verification side and is mentioned in here. 
The nodes returned as part of the `proof.nodes` originally serve as normal Merkle inclusion proof for the proof range i.e., `[proof.start, proof.end)`. Additionally, we perform a **completeness** check on them to ensure that no leaf matching the target namespace is left uncovered by the proof range.  Does this address your question?

Reply:
The completeness property and the intuition behind it are explained earlier in the specs i.e., https://github.com/celestiaorg/nmt/blob/master/docs/spec/nmt.md#namespace-proof-verification